src/dynapyt/runtime.py

```diff
@@ -9,11 +9,8 @@ def set_analysis(new_analysis):
     analysis = new_analysis
 
 def call_if_exists(f, *args):
-    try:
-        func = getattr(analysis, f)
-        return func(*args)
-    except AttributeError:
-        return
+    func = getattr(analysis, f, lambda *args: None)
```

Comment:
I didn't find a good description of why you need to have the lambda function as default. Maybe you can add some comments explaining this in the code for future references.